src/components/EnterBarcodePage/EnterBarcodePage.tsx

```diff
@@ -38,6 +38,7 @@ const EnterBarcodePage = ({
         history.replace(`/patients/new/${ccpID}/${barcode}`);
       }
     }
+    history.replace(`/patients/new/${ccpID}/${barcode}`);
```

Comment:
why was this line added?

Reply:
it's to handle an edge case where the backend wasn't up, or if things weren't loaded in yet